data/json/snippets/mi-go_anatomy.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "type": "snippet",
+    "category": "<mi-go_harvest>",
+    "text": [
+      "As you starts to cut through the thick skin if the mi-go, you note that it's tougher than you would have expected.",
+      "The smell emanating from the mi-go as you cut it open is pungent and very alien.  You're forced to take short breaks to recover from nausea.",
+      "You note that the blood of this creature is unlike any other type of blood you've ever encountered or even heard of.  It's not only the odd color, but also its viscosity and somewhat oily and sticky properties.",
+      "Nothing seems right with this creature.  The texture of the muscles is off.",
+      "You note that the bones in this body have alien shapes and are joined up in ways unlike anything from Earth.",
+      "What should go for tendons in this creature aren't joined to the bones in any way you've seen before.  Instead of a single tendon connecting two end points, there are instead a range of smaller connections between slightly different locations."
```

Comment:
```suggestion
      "What should go for tendons in this creature aren't joined to the bones in any way you've seen before.  Instead of a single tendon connecting two end points, there are instead a range of smaller connections between slightly different locations.  On further study you wonder if these are indeed tendons, or perhaps some kind of nerve or signalling structure.  Later, you find similar things wrapped around what you think might be a sensory organ."
```